students/K_Shaffer/mailroom/mailroom02.py

```diff
@@ -0,0 +1,60 @@
+'''
+Script: mailroom
+Date: 10/28/2019
+Dev: Kory Shaffer
+'''
+
+
+donors = {'William Gates, III': (653784.49, 2),  'Mark Zuckerberg': (16396.10, 3), 'Jeff Bezos': (877.33, 1),
+          'Paul Allen': (708.42, 3)}
```

Comment:
The value should be a list, indicating all donations, and not the summary information about the donations.